### PR TITLE
Support for PHP 7.1

### PR DESCRIPTION
**Pull request by prolic**
- add support for PHP 7.1
- uses custom prooph/common branch ("prooph/common" : "dev-php_7_1 as 4.0")
- uses custom prooph/event-store branch ("prooph/common" : "dev-php_7_1 as 7.0")
- add docheader
- add new method add new method DoctrineEventStoreAdapter::getSqlSchemaFor
- change method signature of DoctrineEventStoreAdapter::createSchemaFor
- replay in chunks (resolves: https://github.com/prooph/event-store-doctrine-adapter/issues/35)
- make replay chunk size configurable
- allow multiple event stores in config

### CODE REVIEW

**Comment by prolic** [comment]
Question: is 100 a good chunk size or should it be increased?

File: composer.json [hunk 1]
```diff
@@ -5,18 +5,17 @@
...
             "email": "contact@prooph.de",
             "homepage": "http://www.prooph.de/"
         },
         {
             "name": "Sascha-Oliver Prolic",
             "email": "saschaprolic@googlemail.com"
+        },
+        {
+            "name": "Bas Kamer",
+            "email": "bas@bushbaby.nl"
```
Comment (basz) [inline composer.json]: Could you change this `baskamer at gmail.com` for public as I get too much spam

**Comment by prolic** [comment]
Ok

Am 27.10.2016 15:26 schrieb "Bas Kamer" <notifications@github.com>:

**Comment by prolic** [comment]
won't fix - use pdo-event-store for v7 series.

**Pull request closed without merging**